internal/cryptogen/msp/msp_test.go

```diff
@@ -110,10 +112,10 @@ func testGenerateLocalMSP(t *testing.T, nodeOUs bool) {
 	}
 
 	tlsCA.Name = "test/fail"
-	err = msp.GenerateLocalMSP(testDir, testName, nil, signCA, tlsCA, msp.CLIENT, nodeOUs)
+	err = msp.GenerateLocalMSP(testDir, testName, nil, signCA, tlsCA, msp.CLIENT, nodeOUs, ECDSA)
```

Comment:
I see that you didn't changed the MSP (top level `msp package`).

The MSP needs to be versioned like [here](https://github.com/hyperledger/fabric/blob/main/msp/mspimpl.go#L112-L139) from the aforementioned reasons.

Reply:
Thanks for all your comments! I intend to add more work to it during the following weeks.